packages/bz3.rb

```diff
@@ -0,0 +1,51 @@
+require 'package'
+
+class Bz3 < Package
+  description 'bzip3 is a better and stronger spiritual successor to bzip2.'
+  homepage 'https://github.com/kspalaiologos/bzip3'
+  @_ver = '1.1.6'
+  version @_ver
+  license 'LGPL-3'
+  compatibility 'all'
+  source_url 'https://github.com/kspalaiologos/bzip3.git'
+  git_hashtag @_ver
+
```

Comment:
Do you think you can add the binaries with @satmandu's scripts?  To upload and add the binaries, see `tools/gl.sh`.

Reply:
I don't have a key and don't seem to have permission to make one

Reply:
I DM'd a deploy token in Slack.